src/libs/feature-previews-config.ts

```diff
@@ -126,6 +127,16 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
     )}`,
     lastUpdated: '11/19/2024',
   },
+  {
+    id: AUTO_GENERATE_DATA_TABLES,
+    title: 'Autogenerate data table for single and paired end sequencing',
+    description:
+      'Enabling this feature will show a new option in the data uploader to autogenerate a data table instead of uploading your own TSV. This feature will attempt to automatch known file patterns for single and paired end sequencing and autogenerate a data table linking to those files.',
+    feedbackUrl: `mailto:dsp-core-services@broadinstitute.org?subject=${encodeURIComponent(
+      'Feedback on Autogenerate data table for single and paired end sequencing'
+    )}`,
+    lastUpdated: '11/26/2024',
```

Comment:
Just curious - is this date a guess for when it's going out or what?

Reply:
Yes, it is a guess of when I anticipate this work to get to prod.